### PR TITLE
Link navigator

### PR DESCRIPTION
**Pull request by deathbeam**
- Add link browser utility service that will try to open specified link in
browser, and in case operation fails displays message box and copy URL
to clipboard.
- Use new LinkBrowser API in SessionManager for opening the links.

Fixes: #434 

Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

Preview:
message

### CODE REVIEW

**Comment by deathbeam** [comment]
See #767 and #777 why this is needed. Removes code duplicity and shows some message to users in case opening of the link fails.

File: runelite-client/src/main/java/net/runelite/client/util/LinkBrowser.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+		try
+		{
+			desktop.browse(new URI(url));
+			log.debug("Opened browser to {}", url);
+			return true;
+		}
+		catch (IOException | URISyntaxException ex)
+		{
+			log.warn("Unable to open specified url", ex);
+			showMessageBox("Unable to open specified url. Link was copied to your clipboard.", url);
```
Comment (devLotto) [inline runelite-client/src/main/java/net/runelite/client/util/LinkBrowser.java]: The user does not know what URL is "specified". Maybe "Unable to open a URL. Link was copied to your clipboard." is better.

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/util/LinkBrowser.java]: Ah, right, true.

File: runelite-client/src/main/java/net/runelite/client/util/LinkBrowser.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	{
+		final ClientUI clientUI = clientUIProvider.get();
+
+		if (clientUI == null)
+		{
+			return;
+		}
+
+		final StringSelection stringSelection = new StringSelection(data);
+		Toolkit.getDefaultToolkit().getSystemClipboard().setContents(stringSelection, null);
```
Comment (SomeoneWithAnInternetConnection) [inline runelite-client/src/main/java/net/runelite/client/util/LinkBrowser.java]: Please don't fuck with my clipboard if I haven't asked you to. Give me a button to press at least

**Comment by Adam-** [comment]
Does this need rebased for #767 ?

**Pull request merged**